pupil_src/shared_modules/time_sync_spec.md

```diff
@@ -1,126 +0,0 @@
-# Pupil Time Sync Protocol
```

Comment:
Maybe we misunderstood each other, I thought we leave this here but link from Pupil helpers to here. (have the spec stay with implementation.)